src/codec2_pitch.c

```diff
@@ -59,19 +32,6 @@ struct CODEC2_PITCH_S {
     COMP          W[FFT_ENC];	        /* DFT of w[] */
 };
```

Comment:
I noticed some warnings when building - think `W` should be a float, e.g. 
``` 
void make_analysis_window(C2CONST *c2const, codec2_fft_cfg fft_fwd_cfg, float w[], float W[]);
```

Also noticed some other unused variable warnings - but we should probably work on these in the codec2 repo.

Reply:
OK, codec2_pitch.c warnings have been resolved. There are a few more in LPCNet itself but that could be a separate PR if desired:

```
[  6%] Building C object src/CMakeFiles/lpcnetfreedv.dir/pitch.c.o
/Users/mooneer/devel/LPCNet/src/pitch.c:229:37: warning: performing pointer subtraction with a null pointer has undefined behavior [-Wnull-pointer-subtraction]
   celt_assert((((unsigned char *)_x-(unsigned char *)NULL)&3)==0);
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mooneer/devel/LPCNet/src/arch.h:63:34: note: expanded from macro 'celt_assert'
#define celt_assert(cond) {if (!(cond)) {celt_fatal("assertion failed: " #cond);}}
                                 ^~~~
1 warning generated.
...
[ 12%] Building C object src/CMakeFiles/lpcnetfreedv.dir/lpcnet_dump.c.o
/Users/mooneer/devel/LPCNet/src/lpcnet_dump.c:120:9: warning: variable 'E' set but not used [-Wunused-but-set-variable]
  float E = 0;
        ^
1 warning generated.
...
[ 35%] Building C object src/CMakeFiles/dump_data.dir/dump_data.c.o
/Users/mooneer/devel/LPCNet/src/dump_data.c:115:9: warning: variable 'E' set but not used [-Wunused-but-set-variable]
  float E = 0;
        ^
1 warning generated.
```